syntax.php

```diff
@@ -182,4 +187,9 @@ private function _selectPrinter($mode, &$renderer, $data){
         }
         return new nspages_printerSimpleList($this, $mode, $renderer, $data);
     }
+
+    private function _cacheOption() {
+         $cache = $this->getConf("cache");
+         return $cache;
```

Comment:
As suggested above, I guess we could put all the cache-related implementation in a same method.

Moreover, since we're only going to use once the config option, I guess we don't even need to store it in a local variable.